frontend/src/app/card.css

```diff
@@ -0,0 +1,14 @@
+.card {
+    max-width: 640px;
+    width: auto;
+    min-width: 300px;
+    margin: 50px auto;
```

Comment:
Top and bottom card margins should be disabled on mobile. They consume valuable screen real-estate.